bin/pbiviz-new.js

```diff
@@ -62,10 +62,19 @@ if (program.force) {
     ConsoleWriter.warn('Running with force flag. Existing files will be overwritten');
 }
 
+let templateExternalJS = {
```

Comment:
we need to avoid such 'hacks'. let's get this working the right way with modular json files managing it per template.